config/routes.rb

```diff
@@ -47,6 +47,14 @@
   get "/register", to: "users#new"
   post "/register", to: "users#create"
```

Comment:
change post to '/users' instead of '/register' and corresponding route errors through app